docs/_static/alabaster.css

```diff
@@ -30,6 +30,8 @@ div.sphinxsidebar {
     width: 220px;
     font-size: 14px;
     line-height: 1.5;
+    max-height: 100%;
+    overflow-y: auto;
```

Comment:
As done in the equivalent PR to cf-python, it is better to put these in the `docs/source/_static/customise-alabaster.css` rather than the `alabaster.css` theme CSS, which to prevent the muddling of original theme and the customisation of it, should remain aligned with the basic Alabaster theme styling.

Reply:
Thanks Sadie, I didn't spot that!